src/status.cpp

```diff
@@ -0,0 +1,26 @@
+//  Copyright (c) 2014-2017 Andrey Upadyshev <oliora@gmail.com>
+//
+//  Use, modification and distribution are subject to the
+//  Boost Software License, Version 1.0. (See accompanying file
+//  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+
+#include "ppconsul/status.h"
+#include "s11n_types.h"
+
+namespace ppconsul { namespace status {
+
+namespace impl {
+
+	std::string parseLeader(const std::string &json)
```

Comment:
Coul you please put reference to the type name `const std::string& json` to match existing code.